includes/wikia/WikiaUpdater.php

```diff
@@ -319,6 +320,19 @@ public static function migrateRecentChangesIpData( DatabaseUpdater $databaseUpda
 		wfWaitForSlaves();
 	}
 
+	public static function removeUnusedGroups( DatabaseUpdater $databaseUpdater ) {
+		global $IP;
+
+		$databaseUpdater->output( "Cleaning up after unused user groups...\n" );
+
+		$worker = $databaseUpdater->maintenance->runChild(
+			'RemoveUnusedGroups',
+			"$IP/maintenance/wikia/removeUnusedGroups.php"
+		);
```

Comment:
Shouldn't we just run a query here instead of starting a new process for each single wiki?

Reply:
name is misleading, it doesn't spawn a new process - it instantiates the class and calls `execute`, so it's all the same